std/experimental/allocator/building_blocks/stats_collector.d

```diff
@@ -742,3 +742,14 @@ public:
     // calls std.experimental.allocator.common.goodAllocSize
     assert((() pure nothrow @safe @nogc => a.goodAllocSize(1))());
 }
+
+// Test that deallocateAll infers from parent
+@system unittest
```

Comment:
Is there something that prevents marking the whole unittest additionally with `nothrow @nogc`?